standard/expressions.md

```diff
@@ -4393,6 +4686,8 @@ There are two forms of the `is` operator. One is the *is-type operator*, which h
 
 #### 12.12.12.1 The is-type operator
 
+*type* shall not be *nullable_reference_type* or a nullable type parameter.
```

Comment:
Commenting here that we need to update 12.12.12.2 on the pattern matching is to point out that something like this is valid:

```csharp
string? s = SomeMethod();
if (s is string str)
{
    // str is not null
}
```

but, checking if an expression is a nullable reference type is invalid.